javascript.js

```diff
@@ -1,9 +1,242 @@
+  },
+];
+
+function project() {
+  const a = document.getElementById('project-portfolio');
+  for (let i = 0; i < array.length; i += 1) {
+    if (i % 2 === 0) {
+      a.innerHTML += `
+  <section class="works" id="portfolio">
+    <div class="content" id="project1">
+      <img src="${array[i].image}" alt="Work 1" class="content-picture"/>
+      <div class="content-body" id="content-body">
+        <h2 class="content-title">${array[i].heading}</h2>
+        <ul class="layout">
+          <li class="call">${array[i].profession[0]}</li>
+          <li><img src="images/point.svg" alt="." /></li>
+          <li class="profession">${array[i].profession[1]}</li>
+          <li><img src="images/point.svg" alt="." /></li>
+          <li class="year">${array[i].profession[2]}</li>
+        </ul>
+        <p class="job-explanation">
+          ${array[i].description}
+        </p>
+        <ul class="programming">
+          <li class="platform">
+            <a href="#html">${array[i].languages[0]}</a>
+          </li>
+          <li class="platform">
+            <a href="#CSS">${array[i].languages[1]}</a>
+          </li>
+          <li class="platform">
+            <a href="#JS">${array[i].languages[2]}</a>
+          </li>
+        </ul>
+        <button onclick="popup(${array[i].number})" type="button" class="content-view-button">See Project</button>
+      </div>
+    </div>
+  </section>`;
+    } else {
+      a.innerHTML += `
+    <section class="works" id="portfolio">
+    <div class="content" id="project1">
+      <div class="content-body" id="content-body">
+        <h2 class="content-title">${array[i].heading}</h2>
+        <ul class="layout">
+          <li class="call">${array[i].profession[0]}</li>
+          <li><img src="images/point.svg" alt="." /></li>
+          <li class="profession">${array[i].profession[1]}</li>
+          <li><img src="images/point.svg" alt="." /></li>
+          <li class="year">${array[i].profession[2]}</li>
+        </ul>
+        <p class="job-explanation">
+          ${array[i].description}
+        </p>
+        <ul class="programming">
+          <li class="platform">
+            <a href="#html">${array[i].languages[0]}</a>
+          </li>
+          <li class="platform">
+            <a href="#CSS">${array[i].languages[1]}</a>
+          </li>
+          <li class="platform">
+            <a href="#JS">${array[i].languages[2]}</a>
+          </li>
+        </ul>
+        <button onclick="popup(${array[i].number})" type="button" class="content-view-button">See Project</button>
+      </div>
+      <img src="${array[i].image}" alt="Work 1" class="content-picture"/>
+    </div>
+  </section>`;
+    }
+  }
+  // a.appendChild();
```

Comment:
- [ ] Please your codebase has a lot of commented lines of code which is not good according to best practises, kindly look through carefully and remove them. 👍